### ISSUE TITLE
beta.16: pipe arguments changed

### ISSUE THREAD

**Issue by gfrSE** [FuelInteractive/fuel-ui#49]
Update for beta.16: [breaking changes](https://github.com/angular/angular/blob/master/CHANGELOG.md#breaking-changes)

```
transform(input: any, ...config: string[]): any {

        if (config.length === 0) {
            config = ["+"];
        }
```

**Comment by coryshaw1** [comment]
I've been working on the fixes for the new Angular 2 update on my repo https://github.com/coryshaw1/fuel-ui

Here's a list of what is broken, and what I've fixed so far:
- [x] Updated dependencies
- [x] LocationStrategy and HashLocationStrategy moved to "angular2/platform/common"
- [x] PipeTransform implementation's parameter changes (this broke Format, OrderBy, and Range)
- [x] CodeHighlighter not properly initializing on view load
- [x] Tag Demo breaking in template
- [x] DatePickers' templates are not updating from component, can't seem to figure this one out yet

Hopefully, we'll be able to figure out the DatePicker problems by the end of the day

**Comment by coryshaw1** [comment]
This is closed with #54
